PULL_REQUEST_TEMPLATE

```diff
@@ -20,5 +20,6 @@
 #### Screenshots (if appropriate)
 #### Checklist:
 - [ ] Documentation Updated
+- [ ] Change log Updated
```

Comment:
I can. I think you _could_ update the change log without needing to update docs, but probably shouldn't. Until I can come up with a valid example, I'll make it a sub- bullet.